### PR TITLE
Qa canonizer sai

### PR DESCRIPTION
**Pull request by saideekshith8**

### CODE REVIEW

File: CanonizerRegistrationPage.py [hunk 1]
```diff
@@ -86,7 +86,9 @@ def create_registration_page_without_entering_in_one_mandatary_field(self, first
         return CanonizerRegisterPage(self.driver)
 
     def registration_title_identification(self):
+        self.click_register_button()
         self.hover(*RegisterPageIdentifiers.TITLE)
+        sleep(3)
```
Comment (akuks) [inline CanonizerRegistrationPage.py]: Why is sleep used here?

File: CanonizerRegistrationPage.py [hunk 2]
```diff
@@ -108,26 +110,72 @@ def verify_firstname_empty(self, Name):
 
     def verify_all_fields_empty(self):
         self.click_register_button()
-        sleep(2)
-        self.find_element(*RegisterPageIdentifiers.Register_now).click()
+        self.find_element(*RegisterPageIdentifiers.REGISTER_NOW).click()
         self.hover(*RegisterPageIdentifiers.error)
         err = self.find_element(*RegisterPageIdentifiers.error).text
         print(err)
         if err == 'Please input your first name!':
```
Comment (akuks) [inline CanonizerRegistrationPage.py]: Please move 'Please input your first name' in config files

**Review by akuks — CHANGES_REQUESTED** [review]

**Comment by saideekshith8** [comment]
Hi, Sleep is used because the test runs faster so user cannot see if test
case is executed or not so to see if the steps are executed we use sleep

I'll add First name in Config File

File: Identifiers.py [hunk 3]
```diff
@@ -22,28 +22,70 @@ class HomePageIdentifiers(object):
     WHATISCANONIZERHEADING = (By.XPATH, '//*[@id="exampleAccordion"]/ul[1]/li[1]/a/span')
     CANONIZER_LOGO = (By.XPATH, '//*[@id="__next"]/div/header/div[1]/a/span/img')
 
+
 class RegisterPageIdentifiers(object):
```
Comment (akuks) [inline Identifiers.py]: Create a New Folder "Name" Identifiers and create new classes there. It'll sort out the code and make it easier to manage

File: Identifiers.py [hunk 4]
```diff
@@ -22,28 +22,70 @@ class HomePageIdentifiers(object):
...
+    FORGET_PASSWORD = (By.XPATH, '//a[text()= "Forgot password"]')
+    TITTLE1 = (By.XPATH, '//h2[text()= "Forgot your password?"]')
+    REGISTER_NOW_LINK = (By.XPATH, '//*[@id="login_form"]/span/a')
+    REQUEST_CODE = (By.XPATH, '//*[@id="login_form"]/div[4]/div/div/div/button[2]/span')
+    EMAIL_ERROR_MESSAGE = (By.XPATH, '//*[@id="login_form"]/div[1]/div[2]/div[2]/div')
+    LOGIN_TITTLE = (By.XPATH, '//h2[text() = "Log in to Canonizer"]')
+    SOCIAL_LINKS = (By.XPATH, '//div[@class="social-login_btn_group__BQdOr"]//child::button')
 
 
+class AccountSettingPageIdentifier(object):
```
Comment (akuks) [inline Identifiers.py:67]: Same as previous comment

**Review by akuks — CHANGES_REQUESTED** [review]

**Comment by akuks** [comment]
@saideekshith8 Please close this PR.

**Comment by saideekshith8** [comment]
The PR is closed, please check ***@***.***

**Comment by akuks** [comment]
No it is not closed. I can see it open (@saideekshith8 )

**Pull request closed without merging**